packages/web/components/Technology/Details/Tabs/Review/index.js

```diff
@@ -55,7 +56,11 @@ const Review = () => {
 
 	return (
 		<Layout.Cell>
-			<Section title="Relatos" hideWhenIsEmpty={false}>
+			<Section
+				title="Relatos"
+				hideWhenIsEmpty={false}
+				rating={calculateRatingsAverage(reviews)}
+			>
```

Comment:
See the comment above ☝️.
You will only need to change RenderProp to a better prop name 🤣
 
```suggestion
			<Section
				title="Relatos"
				hideWhenIsEmpty={false}
				renderProp={() => rating && (
						<RatingText aria-label="Média de Avaliações">
							Avaliação dessa tecnologia: <strong>{rating}</strong>{' '}
							<Rating value={rating} size={3} readonly />
						</RatingText>
					)
                }
			>
```